src/services/bcrypt/index.js

```diff
@@ -0,0 +1,41 @@
+import { logger } from "#utils/logger/logtail.js";
+import { getCommandArgs } from "#utils/command.js";
+import * as bc from "bcrypt";
+
+/**
+ *
+ * @param {import('telegraf').Context} context
+ * @returns {Promise<void>}
+ */
+async function bcrypt(context) {
+  const text = getCommandArgs("bcrypt", context);
+  const salt = bc.genSaltSync(10);
+  const hashedText = bc.hashSync(text, salt);
```

Comment:
are there any reasons behind the use of the sync version instead of its async counterpart?

Reply:
just to write simple code

Reply:
I'm not sure if `bc.genSaltSync(10)` is any simpler than just doing `await bc.genSalt(10)`

Reply:
got it, i'll update it to use `await bc.genSalt(10)`